ds_judgements_public_ui/sass/includes/_help_end_document_marker.scss

```diff
@@ -0,0 +1,55 @@
+.help-end-document-marker {
```

Comment:
This class only slightly differs from `.judgment-end-document-marker ` with the differences shown in the screenshot:

![Screenshot 2023-04-06 at 11 31 30](https://user-images.githubusercontent.com/42998618/230352208-8b55f92f-3179-4b4a-b97b-a058bbe6eba9.png)

Are these differences intentional?

If so can we refactor so that we reduce duplication, otherwise can we drop one of them?

Reply:
aha yep, the stylistic difference is on purpose, but yep, agreed on cleaning up the definitions. I'll take a look!